packages/styletron-react/package.json

```diff
@@ -12,7 +12,7 @@
   ],
   "main": "./dist/index.js",
   "module": "./dist/index.es.js",
-  "types": "./lib/index.d.ts",
```

Comment:
this might is a bit tricky one - for editing experience and in particular for project references to work correctly, it might need to point to lib with compilation results

Reply:
what do you mean? We are just keeping them in `types: "./dist/index.d.ts"`

Reply:
the difference would be only when changing something in referenced module, for example when editing `styletron-standard` with `types: lib/index` that definition file would be automatically recompiled when doing type check in `styletron-react` or other project depending on it… 

but the bundle in dist folder would not be recompiled automatically by typescript